python/lsst/jointcal/ccdImage.cc

```diff
@@ -66,13 +65,11 @@ void declareCcdImageList(py::module &mod) {
 
     cls.def("sizeValidForFit", &CcdImageList::sizeValidForFit);
 
-    // Make this behave like a std::list, which it is, but we can't wrap children of it easily.
+    // Make this behave like a std::list (which it is but we can't wrap std::list subclasses trivially)
```

Comment:
One constraint is that MeasuredStar's contain some reference to their mother CcdImage.
For sake of efficiency, this is implemented as some sort of pointer, and so  CcdImage's should not move in memory. The no-copy policy of CcdImages is enforced by making the copy constructor private (the comment there does not tell why. My fault). I am guessing this was the original constraint, but maybe at some point, I switched to a container of pointers for the CcdImageList, which voids the argument. So, it might just work as well with std::vector as a container. But unless a new scheme is devised to represent the MeasuredStar->CcdImage relation, copying CcdImage's should remain forbidden, which excludes a std::vector<CcdImage>.